examples/transformerconfigs/README.md

```diff
@@ -1,37 +1,44 @@
 # Transformer Configurations
 
-Kustomize computes the resources by applying a series of transformers:
+Kustomize computes the configuration of a resource by applying a series of transformers:
```

Comment:
```
Kustomize creates new resources by applying
a series of transformations to an original
set of resources.
```

Reply:
OK. How to differentiate between resources defined in manifests and live resources/object?

Reply:
What about this possible change to below:

Kustomize provides the following default transformers:

- namespace
- prefix/suffix
- label
- annotation
- name reference
- variable reference